src/styles/button.css

```diff
@@ -0,0 +1,21 @@
+/* #region button */
+.product-card__buy-button {
+  text-align: center;
+  height: 40px;
+  width: 100%;
+  background: #00acdc;
```

Comment:
more correctly move all your colors to variables(not required but would be cool)

Reply:
I've done everything, but I'm not sure if I used the right selector for the color variable.